src/aics-image-viewer/components/App/types.ts

```diff
@@ -1,8 +1,48 @@
 import { View3d, Volume, ImageInfo } from "@aics/volume-viewer";
-import { ImageType, ViewMode } from "../../shared/enums";
+import { ImageType, RenderMode, ViewMode } from "../../shared/enums";
+import { PerAxis } from "../../shared/types";
 import { ColorArray } from "../../shared/utils/colorRepresentations";
 import { ChannelGrouping, ChannelState, ViewerChannelSettings } from "../../shared/utils/viewerChannelSettings";
 
+type ControlNames =
+  | "alphaMask"
```

Comment:
probably should be `alphaMaskSlider` for consistency? Or consider removing the suffix from all the others, to be agnostic to the control type?